src/overlay/PendingEnvelopes.cpp

```diff
@@ -144,7 +142,7 @@ PendingEnvelopes::isNodeInQuorum(NodeID const& node)
     else
     {
         // search through the known slots
-        SCP::TriBool r = mHerder.getSCP().isNodeInQuorum(node);
+        SCP::TriBool r = mApp.getHerder().getSCP().isNodeInQuorum(node);
```

Comment:
don't move quorum related functions to overlay: we should keep overlay agnostic to SCP as much as possible